### ISSUE TITLE
Geth 5.4.3 not syncing

### ISSUE THREAD

**Issue by ciprianrpp** [ethereumproject/go-ethereum#645]
Hello guys,
I've downloaded the latest release and started on the mainnet with: ./geth --rpc console
Is stuck forever, the sync doesn't start.
I think it has something to do with the NAT mapping. (it shows mapping instead of Mapped)
I've tested with ethereum blockchain also and the mapping & sync works (same ports).
Please advice.
Thank you.
`
2018-07-11 11:21:38 Geth Classic version: v5.4.3-5291a28
2018-07-11 11:21:38 Blockchain: Ethereum Classic Mainnet
2018-07-11 11:21:38 Chain database: /home/pool/.ethereum-classic/mainnet/chaindata
2018-07-11 11:21:38 Blockchain upgrades configured: 6
2018-07-11 11:21:38  1150000 Homestead
2018-07-11 11:21:38  1920000 The DAO Hard Fork (0x94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f)
2018-07-11 11:21:38  2500000 GasReprice
2018-07-11 11:21:38  3000000 Diehard
2018-07-11 11:21:38  5000000 Gotham
2018-07-11 11:21:38  5900000 Defuse Difficulty Bomb
2018-07-11 11:21:38 Using 10 configured bootnodes
2018-07-11 11:21:38 Use Sputnik EVM: false
2018-07-11 11:21:38 No etherbase set and no accounts found as default
2018-07-11 11:21:39 Allotted 1024MB cache and 1024 file handles to /home/pool/.ethereum-classic/mainnet/chaindata
2018-07-11 11:21:46 Allotted 16MB cache and 16 file handles to /home/pool/.ethereum-classic/mainnet/dapp
2018-07-11 11:21:46 Protocol Versions: [63 62], Network Id: 1, Chain Id: 61
2018-07-11 11:21:47 Genesis block: 0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3 (mainnet)
2018-07-11 11:21:47 Local head header:     #846258 [0x6321c7…] TD=5517582828633640920
2018-07-11 11:21:47 Local head full block: #846258 [0x6321c7…] TD=5517582828633640920
2018-07-11 11:21:47 Local head fast block: #846258 [0x6321c7…] TD=5517582828633640920
2018-07-11 11:21:47 Starting server...
2018-07-11 11:21:47 Mapping network port udp:30303 -> 30303 (ethereum discovery) using UPNP IGDv1-IP1
2018-07-11 11:21:47 UDP listening. Client enode: enode://e578cb56c182550fc0f46c1209994f5f62e871aa5d335131d3aea2dc1dc242be6c5a6e9dd8bfb76cd56d5d5e892878781547cb6b1e8ca2c7506b45b13f4a92b2@172.31.70.28:30303
2018-07-11 11:21:47 HTTP endpoint: http://localhost:8545
2018-07-11 11:21:47 IPC endpoint opened: /home/pool/.ethereum-classic/mainnet/geth.ipc
2018-07-11 11:21:47 Mapping network port tcp:30303 -> 30303 (ethereum p2p) using UPNP IGDv1-IP1
Welcome to the Geth JavaScript console!

instance: Geth/v5.4.3-5291a28/linux/go1.9.7
 modules: admin:1.0 debug:1.0 eth:1.0 geth:1.0 miner:1.0 net:1.0 personal:1.0 rpc:1.0 txpool:1.0 web3:1.0

>
`

**Comment by whilei** [comment]
Just to cover some bases; it looks like your in geth's interactive `javascript console`, so it's possible you won't see logs like normal. Have you tried the `eth.syncing` and/or `admin.peers` command there? (It looks like you have block sync progress through 846258, though maybe that's from an earlier run with a different release?)

If this doesn't answer or help, can you please attach your debug logs from when this happens? You'll find them organized by time in `~/.ethereumc-classic/mainnet/logs/`.

**Comment by tzdybal** [comment]
I can confirm, that in console, you won't see the logs, but the sync will progress. Give it some time, as you need to connect to some peers first. Check `eth.blockNumber` just after starting the console, and 15 minutes later. The number should increase.

**Comment by ciprianrpp** [comment]
Ok guys.
I've deleted ~/.ethereumc-classic to start fresh with this new geth version.
I've started with verbosity 6. 
The console logs for importing do not appear (like on older version of geth), but the sync is going.(hope it will be fixed on next versions)
`> eth.syncing
{
  currentBlock: 53667,
  highestBlock: 5944548,
  knownStates: 0,
  pulledStates: 0,
  startingBlock: 0
}
>
> eth.blockNumber
58436
>

`
